ethpm/exceptions.py

```diff
@@ -1,12 +1,20 @@
 class InsufficientAssetsError(Exception):
```

Comment:
@njgheorghita @davesque not for this PR but I think `ethpm` should follow the same strategy as py-evm/trinity in that all exceptions inherit from some common application base exception (e.g. `BaseEthPMException`)

https://github.com/ethereum/py-evm/blob/82022a78a8b2419fc090af21428f20e2c01172f4/trinity/exceptions.py#L4